tests/test_schemas.py

```diff
@@ -446,17 +446,17 @@ def test_schema_for_regular_views(self):
                             coreapi.Field('ordering', required=False, location='query', schema=coreschema.String(title='Ordering', description='Which field to use when ordering the results.'))
                         ]
                     ),
-                    'custom_list_action': coreapi.Link(
+                    'get_1': coreapi.Link(
```

Comment:
Can we avoid all these renames?

Reply:
Surely not every time because that is a part of the issue, they clash sometimes. Are they used anywhere?

Reply:
They're used in the interactive API documentation. 

Can we keep them as are if there is no collision and only append a number when there is more than one? 

i.e. here, if there were a collision, we'd have `custom_list_action` and `custom_list_action_1` and so on.

Reply:
We can do it